includes/admin_import.php

```diff
@@ -542,9 +704,37 @@ function tsml_import_page()
 						<?php } ?>
 						<p><?php printf(__('Want to send a mass email to your contacts? <a href="%s" target="_blank">Click here</a> to see their email addresses.', '12-step-meeting-list'), admin_url('admin-ajax.php') . '?action=contacts') ?></p>
 					</div>
+
+					<!-- Data Sources Debug Mode -->
+					<div class="postbox stack">
+						<h2><?php _e('Data Sources Debug Mode', '12-step-meeting-list') ?></h2>
+						<?php
+						if ($meetings) { ?>
+							<form class="stack compact" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+								<h3><?php _e('Change Detection', '12-step-meeting-list') ?></h3>
+								<p><?php printf(__('Turn test_mode on to see first change detected on an import record.', '12-step-meeting-list')) ?></p>
+								<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+								<select name="tsml_detection_test_mode" onchange="this.form.submit()">
+									<?php
+									foreach ([
+										'on' => __('On', '12-step-meeting-list'),
+										'off' => __('Off', '12-step-meeting-list'),
+									] as $key => $value) { ?>
+										<option value="<?php echo $key ?>" <?php selected($tsml_detection_test_mode, $key) ?> >
+											<?php echo $value ?>
+										</option>
+									<?php } ?>
+								</select>
+							</form>
+						<?php } ?>
+					</div>
 				</div>
 			</div>
 		</div>
+	</div>
+</div>
```

Comment:
Removing this </div> will fix the issue with the footer overlapping the page.

Reply:
Thanks. I was thinking we should remove the footer altogether.